membership_extension/tests/test_membership.py

```diff
@@ -10,16 +10,25 @@
 
 from odoo import fields
 from odoo.exceptions import UserError, ValidationError
-from odoo.tests import common
+from odoo.tests import common, tagged
 from odoo.tools import mute_logger
 
 
-@freeze_time("2025-01-01")
+@tagged("-at_install", "post_install")
```

Comment:
Why unfreezing? 🤔

Reply:
This is necessary to avoid side effects:
- If another module inherits from this one the tests will be blocked.
- If the test is in post_install, all the tests that are executed will use that date.

Some references:
- https://github.com/OCA/credit-control/pull/434#issuecomment-2690137011
- https://github.com/OCA/field-service/pull/1340#issuecomment-2711247458